### ISSUE TITLE
Credit not reflected in account?

### ISSUE THREAD

**Issue by ekimpossible** [ssraza21/textifyi-issues#1]
Hi! I just recently found out about textify to convert an arabic pdf and purchased 300 credits. However, the credits don't seem to reflect in my account--it only shows that I have 10. Could you look into it? Thanks!

**Comment by ssraza21** [comment]
Looking into it!

**Comment by ssraza21** [comment]
Sorry, is your account i***********tter925@gmail.com?

It seems like the payment was done through a different account and you were logged in textifyi in another. 

I've given you the credits but please confirm that is indeed your email! And sorry for the inconvenience. If that it not, I can update your actual account

**Comment by ekimpossible** [comment]
Hi! Thanks for responding so fast ◡̈ Yes, that is my account! Thanks for your help and for making this software accessible!


Warmly,

Esther Kim

________________________________

**Comment by ssraza21** [comment]
Of course! Let me know if you run into any other issues and I'll be happy to help asap

**Comment by ssraza21** [comment]
Is it okay if I react out to you through your university email? Would love to get your insights on how you can see the software improving, etc.

**Comment by ekimpossible** [comment]
Of course! Go ahead ◡̈


Warmly,

Esther Kim

________________________________
